tests/functional_tests_Sampler_only.bash

```diff
@@ -31,6 +81,51 @@ function Functional_Test__do-Sampler-only()
     fi
     Check_If_Software_Produced_Expected_Output 'Sampler' "$(pwd)/Sampler"
     mv 'Sampler' 'Sampler-success'
+    # Expect success with FIST module
+    mkdir -p "Sampler/${run_id}"
+    touch "${particle_list}"
+    touch "${decays_list}"
```

Comment:
Touch the two files in single `touch`. Same applies later at many points in the same file.